angular-portfolio/src/app/awards.service.ts

```diff
@@ -6,16 +6,15 @@ import { HttpClient, HttpHeaders } from '@angular/common/http';
 @Injectable({
   providedIn: 'root'
 })
-
 export class AwardsService {
 
-  private award = '/load-awards';
+  private listAwardsResouce = '/load-awards';
```

Comment:
private static readonly LIST_AWARDS_RESOURCE

This can be static because the string is always the same.

Reply:
If we're making it a constant, I think it would actually make more sense to put this out of the component, and just be "const LIST_AWARDS_RESOURCE".

Doing it with "static" requires it to be referenced as AwardsService.LIST_AWARDS_RESOURCE, which I find a little unwieldy.